src/tabnineChatWidget/extensionCommands/commands.ts

```diff
@@ -0,0 +1,73 @@
+type SCOPE = "block" | "selection" | "none";
```

Comment:
```suggestion
type Scope = "block" | "selection" | "none";
```
?
is this a known type in vscode sdk? I don't see its usage